contents/javascript/Javascript-framework-VueJs.md

```diff
@@ -79,13 +87,163 @@ In this case, only the `root` component can be accessed in VueJs while the rest
 <b>Step 4:</b> Pass the message to the HTML file using double curly brackets.
 ```HTML
   <div id="root">
-    <h2>{{ message }}</h2>
+    <h2>{\{message}\{</h2>
   </div>
 ```
 
 <b>Step 5:</b> Open the brower and we will see "Hello World" being displayed:
 > <h2>Hello World</h2>
+<br>
```

Comment:
I know this example is from the original version and there is no need for you to fix it. Nevertheless, the reader will wonder "why go into all this trouble to display Hello World?".